packages/hardhat-core/src/builtin-tasks/compile.ts

```diff
@@ -675,8 +675,16 @@ subtask(TASK_COMPILE_SOLIDITY_RUN_SOLC)
   .addParam("input", undefined, undefined, types.any)
   .addParam("solcPath", undefined, undefined, types.string)
   .setAction(
```

Comment:
This is missing the new param definition in the task. I think it's something like:

```
.addOptionalParam("solcVersion", undefined, undefined, types.string)
```